runtime/jextractnatives/jextractnatives.c

```diff
@@ -289,29 +289,6 @@ cacheIDs(JNIEnv* env, jobject dumpObj)
 	return 0;
 }
 
-void JNICALL
```

Comment:
Line 28
```c
#include "jextractnatives_internal.h"
```
must be deleted; it refers to a file deleted by this change.

Also, please don't leave two consecutive blank lines here.